deploy/operator/setup_assisted_operator.sh

```diff
@@ -76,6 +76,12 @@ cat <<EOF
   PUBLIC_CONTAINER_REGISTRIES: 'quay.io,${PUBLIC_CONTAINER_REGISTRIES}'
 EOF
     fi
+    if [ -n "${ENABLE_CONVERGED_FLOW:-}" ]; then
+cat <<EOF
+  ENABLE_CONVERGED_FLOW: '${ENABLE_CONVERGED_FLOW}'
```

Comment:
how exactly should that work?
isn't it ``ALLOW_COVERGED_FLOW``?

Reply:
Thanks